flight_busiest_traveling_period/v1/get/Java SDK/FlightBusiestPeriod.java

```diff
@@ -16,7 +16,7 @@ public static void main(String[] args) throws ResponseException {
     // Flight Busiest Traveling Period
     Period[] busiestPeriods = amadeus.travel.analytics.airTraffic.busiestPeriod.get(Params
       .with("cityCode", "MAD")
-      .and("period", "2017")
```

Comment:
This should not change, as the API takes as input dates in the past.

Reply:
Resolved the issue.